core/templates/services/context.service.spec.ts

```diff
@@ -266,5 +315,15 @@ describe('Context service', () => {
       expect(ecs.getPageContext()).toBe('other');
     }
     );
+
+    it('should retrive exploration preview mode', () => {
+      expect(ecs.getEditorTabContext()).toBe('preview');
+    });
+
+    it('should retrieve entity id and type', () => {
+      ecs.setCustomEntityContext('other', '100');
```

Comment:
Should there be a precheck e.g. is customEntityContext a different value before this?

Reply:
Done!